Dockerfile.redhat

```diff
@@ -285,7 +285,7 @@ COPY yarn.lock /ovms/
 COPY package.json /ovms/
 
 # prebuild dependencies before copying sources
-RUN bazel build --jobs=$JOBS ${debug_bazel_flags} //:ovms_dependencies
+RUN bazel build --jobs=$JOBS ${debug_bazel_flags} //:ovms_dependencies @com_google_googletest//:gtest
```

Comment:
Yes, this allows us to not rebuild whole ovms_test each time, as whole ovms_test depend on gtest. Until it is fetched build will not start. While this will make RUN_TESTS=0 option still download gtest, this will also allow us to better use docker cache.